src/index.ts

```diff
@@ -31,6 +31,14 @@ type MakeUniversalOpts = {
    * Forcefully overwrite any existing files that are in the way of generating the universal application
    */
   force: boolean;
+  /**
+   * Fuse x64 and arm64 ASARs into one.
+   */
+  fuseASARs?: boolean;
```

Comment:
Can we not use the word "fuse" here as fuses are a different Electron concept.

Either `mergeASARs` or `lipoNativeModules` might be better options here.  The latter definition if we remove the allow list option below because then that's all this option does.

Reply:
Renamed to `mergeASARs`, which is frankly what I came up originally :joy: